src/ATDPApplication.ts

```diff
@@ -126,6 +137,28 @@ export abstract class ATDPApplication<T> extends ACLUEWrapper {
     });
   }
 
+  /**
+   * Loads the client config from '/clientConfig.json' and parses it.
+   */
+  public static loadClientConfig<T = any>(): Promise<T | null> {
+    return Ajax.getJSON('/clientConfig.json').catch((e) => {
+      // TODO: Do you want to print an error here, or should it fail silently?
+      console.error('Error parsing clientConfig.json', e);
```

Comment:
I would log an error and add empty clientConfigs.json to every existing apps. New apps should have an empty _clientConfig.json_ by default.

We should probably remove the TODO marker before merging the PR.